.coveragerc

```diff
@@ -775,6 +775,7 @@ omit =
     homeassistant/components/smappee/binary_sensor.py
     homeassistant/components/smappee/sensor.py
     homeassistant/components/smappee/switch.py
+    homeassistant/components/smart_meter_texas/sensor.py
```

Comment:
```suggestion
```

Can we remove this now?

Reply:
I added it because it doesn't have 100% test coverage. I had trouble getting all functionality to test.

Reply:
In that case, let's leave it in.

Reply:
FYI
```
Name                                                        Stmts   Miss  Cover   Missing
-----------------------------------------------------------------------------------------
homeassistant/components/smart_meter_texas/__init__.py         61      0   100%
homeassistant/components/smart_meter_texas/config_flow.py      48      0   100%
homeassistant/components/smart_meter_texas/const.py            10      0   100%
homeassistant/components/smart_meter_texas/sensor.py           57      3    95%   89, 111-112
-----------------------------------------------------------------------------------------
```

Reply:
You'll get line 89 covered when using the generic entity update service

Reply:
I believe our codecov target is set to 95.4% so you should be able to take it out of `.coveragerc` once the updates are switched over. 😄

Reply:
```
Name                                                        Stmts   Miss  Cover   Missing
-----------------------------------------------------------------------------------------
homeassistant/components/smart_meter_texas/__init__.py         61      0   100%
homeassistant/components/smart_meter_texas/config_flow.py      48      0   100%
homeassistant/components/smart_meter_texas/const.py            10      0   100%
homeassistant/components/smart_meter_texas/sensor.py           57      2    96%   111-112
-----------------------------------------------------------------------------------------
TOTAL                                                         176      2    99%
```